packages/core/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -91,6 +91,7 @@ export class ChoiceList extends React.PureComponent {
         onBlur={this.props.onBlur}
         onChange={this.props.onChange}
         className={classes}
+        aria-label={this.props.ariaLabel}
```

Comment:
@amistein Looking at this again in terms of @bernardwang comment, we may need to add this prop to the `Select` component also. We can render an HTML `<select>` from ChoiceList, but also the standalone Select. Should be a one line add, and give us the flexibility to use proper FormLabel components as the best practice, or `aria-label` for cases where that's not possible.

Reply:
It seems like we are currently able to use `aria-label` with the `<Select>` component directly, since in `<Select.jsx>` we pass `...selectProps` to the HTML `<select>`. Do you want me to add it as an explicit prop?

Reply:
Yeah, you're right @amistein. Since we can pass undocumented props directly into the `<Select />` it's okay to pass on adding `aria-label`. Good call.